### DISCUSSION TITLE
Why does my Pause/Resume not work?

### DISCUSSION THREAD

**Discussion by lrpauley13** [melanchall/drywetmidi#306] [Q&A] [answered]
I wrote a quick little test program for pausing and resuming. Does anyone have an idea as to why it does not work?
[DryWetMidiLightTest2.zip](https://github.com/user-attachments/files/16643255/DryWetMidiLightTest2.zip)

I did not include the MIDI, because it is not supported.  Any Midi can be used to test.

All I want to do is stop playing and resume playing.  The button seems to take you back to the beginning of the Midi.

**Comment by melanchall** [comment, marked as answer]
I've answered [here](https://github.com/melanchall/drywetmidi/discussions/300#discussioncomment-10365987).
